lint-staged.config.js

```diff
@@ -3,6 +3,10 @@
 module.exports = {
   "*.{js,js.flow,ts}": "eslint --fix --report-unused-disable-directives",
   "*.md": "prettier --write",
+  "*.mdx": [
```

Comment:
`.md` can be merged into this one

Reply:
First of all, you're awesome for reviewing this PR. ❤️ 

What do you mean by this, though? We use Prettier through ESLint, so it`*.mdx` doesn't need to be prettified additionally, unlike `*.md` files which _only_ use Prettier.

Reply:
Why `remark-lint` related rules are not enabled for `.md` files?

Reply:
That's just a temporary thing, probably. We were focusing on the new docs that we're adding, which are all `.mdx`. We can then go over the older docs we have and see how the rules and everything works. Didn't want to fix a bunch of errors without knowing if we liked the workflow. 🙂